src/Inkluzitron/Handlers/MessagesHandler.cs

```diff
@@ -58,13 +61,33 @@ static public bool TryParseMessageAndCheck(SocketMessage message, out SocketUser
             return true;
         }
 
-        private bool IsCommand(SocketUserMessage message, ref int argPos)
+        private bool IsCommand(IUserMessage message, ref int argPos)
+            => message.HasMentionPrefix(DiscordClient.CurrentUser, ref argPos) || HasCommandPrefix(message, ref argPos);
+
+        private bool HasCommandPrefix(IUserMessage message, ref int argPos)
+            => message.Content.Length > CommandPrefix.Length && message.HasStringPrefix(CommandPrefix, ref argPos);
+
+        public async Task<SingleCommandMatch> MatchSingleCommandAsync(IUserMessage message)
         {
-            if (message.HasMentionPrefix(DiscordClient.CurrentUser, ref argPos))
-                return true;
+            int argPos = default;
```

Comment:
není tohle to, co dělá už knihovna? :hmmnote:

Reply:
Knihovna mi umí najít příkaz (search) nebo ho rovnou pustit (executeasync). Nenašel jsem ale způsob, jak ji donutit udělat jen tohle (search + parse, ale ne execute).